packages/core/lib/modules/wallet.js

```diff
@@ -61,6 +66,16 @@ class Wallet {
     this.eventManager.registerEvents(WalletEvents);
     this.accounts = Accounts.getInstance({wallet: this});
 
+    global.walletInstances++;
+
+    setTimeout(() => {
+      if (global.walletInstances > 0) {
+        console.warn(
+          "Multiple wallet instances were created. If that's not intentional please check your code, and use the Wallet.getInstance() instead of creating a new instance",
```

Comment:
what are the dangers of multiple wallets? How do I access a specific wallet if multiple are created?

Reply:
Yes that sounds better, i know there are several implications. Just added the warning to prevent bugs, but throwing seems much better